### ISSUE TITLE
Version?

### ISSUE THREAD

**Issue by Chdata** [alongubkin/store#17]
What version of store is the master branch?

What version of store is dev branch?

What's the difference between the master and dev?

What's the difference between this github, and what's shown in the download of

https://forums.alliedmods.net/showthread.php?t=207157

^ This thread being store 1.1-alpha

Sorry if I overlooked something, but I think at least

**Comment by Arrow768** [comment]
Thats a very good question.

The master branch should be 1.1, but several commits have been made to it.
The dev branch should be 1.2(dev), but its missing some commits from master and has got additional commits that are not in master

Then there are the branches from other people, that fix some bugs and introduce a few new features

I guess someone with a decent sourcepawn understanding should take the time and check out if / how master and develop can be merged again. 
It might also be worth to check if some of the other branches can / should be merged into master

**Comment by Chdata** [comment]
Well my fork replaces this <colors> <morecolors_store> nonsense with just <morecolors>, and probably some other minor issues I can't remember. But mines is made to work for me, and for my TF2 servers.

Thus I unfortunately can't guarantee that anything from it will be compatible with any/all of the games store is meant to be compatible with. Especially since I probably don't have a lot of other source games or know how to set up servers for them or have any need/desire to.

I do have a vague idea of what some things might or might not be compatible with all games though... but it's limited.

I'm not sure of all of the differences between the main/dev branch. I use the dev branch and at least know the mains store-gifting.sp is more stable. That's about it.

I also know that the setup for this thing and upgrading between different versions is very confusing XP

tl;dr maybe I can help with small fixes some time? idk
i'm pretty new to github too

**Comment by Arrow768** [comment]
@JackofDesigns (the current maintainer of the store plugin) mentioned that he is going to merge all the Branches once he has finished another project.

**Comment by Chdata** [comment]
Lol, he's a friend of mines. He's always doing a billion things o.O

sadlaughter, which makes it sound like it'll be some time before that happens, he really is quite a busy guy.
